debug-tests

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+gdb --args build/debug/test/tester ion-tests
```

Comment:
No idea? I figure we'll cross the cross-platform bridge when we come to it, just getting the code to compile is a more likely first step than getting debugging working. Or did you mean just add "-gdb" to the file name so that it's clear what's going on?

I'll add the file extensions